### PR TITLE
LPS-103629 Creation of the Metal Component and its template for the new field

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Separator/Separator.es.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	repeatable: Config.bool(),
+
+	/**
+	 * @default true
+	 * @instance
+	 * @memberof Separator
+	 * @type {?(bool|undefined)}
+	 */
+
+	showLabel: Config.bool().value(true),
```
Comment (oliveiraaraujo) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Separator/Separator.es.js]: @marcelabc what do you think to remove undefined from this case?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Separator/Separator.es.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+	/**
+	 * @default undefined
+	 * @instance
+	 * @memberof Separator
+	 * @type {?(string|undefined)}
+	 */
+
+	style: Config.string().value(''),
```
Comment (oliveiraaraujo) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Separator/Separator.es.js]: @marcelabc, what do you think about replace default to ''

**Comment by oliveiraaraujo** [comment]
LGTM!

**Pull request closed without merging**